.gitattributes

```diff
@@ -0,0 +1,2 @@
+# enforce unix style line endings
```

Comment:
Could you pls revert this change? It does not belong to this PR.

Feel free to open a new issue and explain why Windows development requires such a change. Then we can discuss & decide :)